packages/viz/src/lib/style/helpers/utils.ts

```diff
@@ -110,3 +125,15 @@ export function calculateSizeBins(nBuckets: number, sizeRange: number[]) {
   };
   return new Classifier(classObj).breaks(nBuckets, 'equal');
 }
+
+const lerp = (x: number, y: number, a: number) => x * (1 - a) + y * a;
+const clamp = (a: number, min = 0, max = 1) => Math.min(max, Math.max(min, a));
+export const invlerp = (x: number, y: number, a: number) =>
+  clamp((a - x) / (y - x));
```

Comment:
I'm not sure if we still need these functions after the change I made for using chroma.scale. If we still do, IMO we should move them to some math-utils.ts